R/dfm_compress.R

```diff
@@ -74,14 +79,18 @@ dfm_compress.dfmSparse <- function(x, margin = c("both", "documents", "features"
 #' # equivalent
 #' dfm(mydfm, groups = "grp")
 #' dfm(mydfm, groups = c(1, 1, 2, 2))
-dfm_group <- function(x, groups = NULL) {
+dfm_group <- function(x, groups = NULL, fill = FALSE) {
```

Comment:
Here I've added a `fill` argument, but it doesn't do anything (yet) - you will need to review the tests and add the behaviour to match. But first: See comments on the tests below.